### PR TITLE
Locate backend refactoring

### PR DESCRIPTION
**Pull request by lilly-muyama**
1. What does this PR?
This PR created endpoint for the Locate part of airqo netmanager
2. What JIRA tasks are related to this PR?
They are currently inaccessible
3.  How do I test out this PR?
Git clone this repo `cd AirQo-api/src/locate` . Create an nusing python  -m venv env and activate it using .\env\scripts\activate or source\activate for MAC users. Run pip install -r requirements.txt. Afterwards type `flask run` to test out the endpoints using any REST client tool available to you.
4. Which end points should are ready for testing?
Base url: http://127.0.0.1:4001/api/v1
Partial urls:
1. /map/parishes
2. /map/savelocatemap
3. /api/v1/map/getlocatemap/<user_id>
4. /api/v1/map/updatelocatemap/<space_name>
5. /api/v1/map/deletelocatemap/<space_name>

### CODE REVIEW

File: src/locate/app.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from flask import Flask
+import logging
+import os, sys
+from flask_cors import CORS
+from flask_pymongo import PyMongo
```
Comment (danielogen) [inline src/locate/app.py:5]: This line produce an error: `Error: While importing "locate.app", an ImportError was raised:

Traceback (most recent call last):
  File "/anaconda3/lib/python3.7/site-packages/flask/cli.py", line 240, in locate_app
    __import__(module_name)
  File "/Users/danielogenrwot/Documents/Projects/AirQo Platform/AirQo-api/src/locate/app.py", line 5, in <module>
    from flask_pymongo import PyMongo
ModuleNotFoundError: No module named 'flask_pymongo'`

Comment (danielogen) [reply src/locate/app.py:5]: Never mind...  It was an issue with my macOS environment setup.

File: src/locate/app.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+from controllers.locate import locate_blueprint
```
Comment (danielogen) [inline src/locate/app.py:6]: Getting some import error: `ModuleNotFoundError: No module named *controllers*`

**Comment by lilly-muyama** [comment]
> 
> 
> Thanks, the endpoints for locate work as expected. Awesome.
> 
> Please provide a quick PR description. Which miscroservice are we testing?
> Is this PR for 2 microservices? `src/locate` and `src/location-registry`?



> 




> 
> 
> Thanks, the endpoints for locate work as expected. Awesome.
> 
> Please provide a quick PR description. Which miscroservice are we testing?
> Is this PR for 2 microservices? `src/locate` and `src/location-registry`?

Done

**Comment by lilly-muyama** [comment]
This PR is no longer valid

**Pull request closed without merging**